### ISSUE TITLE
Sandbox gives error after adding third type definition

### ISSUE THREAD

**Issue by nematsakis** [amzn/ion-schema#87]
When using the [Ion Schema Sandbox](https://amzn.github.io/ion-schema/sandbox) I am getting an error "Can not load given schema! Please check Ion schema syntax for given schema." whenever I add three types.  For example, if I try to evaluate the value `22` against the schema below, it will work for any of the three with one of the other two types specified, but as soon as I add a third type, I get an error message.

```
type::{
  name: foo,
  type: int,
  valid_values: range::[1, 100],
}

type::{
  name: bar,
  type: int,
  valid_values: range::[1, 100],
}

type::{
  name: baz,
  type: int,
  valid_values: range::[1, 100],
}
```

**Comment by desaikd** [comment]
Hi @nematsakis,
Thanks for reporting this issue! 
It seems like the issue related to how the schema content is being fetched for validation from the schema editor. 
We will work on fix for this and update here once a solution is merged.

**Comment by desaikd** [comment]
This has been fixed with #89
